conan/internal/runner/ssh.py

```diff
@@ -189,26 +191,15 @@ def ensure_runner_environment(self):
         _, _stdout, _stderr = self.client.exec_command(f"{self.remote_conan} config home")
         ssh_info(f"Remote conan config home returned: {_stdout.read().decode().strip()}")
         _, _stdout, _stderr = self.client.exec_command(f"{self.remote_conan} profile detect --force")
-        self._copy_profiles()
+        self._sync_conan_config()
 
-
-    def _copy_profiles(self):
-        sftp = self.client.open_sftp()
-
-        # TODO: very questionable choices here
-        try:
-            profiles = {
-                self.args.profile_host[0]: self.host_profile.dumps(),
-                self.args.profile_build[0]: self.build_profile.dumps()
-            }
-
-            for name, contents in profiles.items():
-                dest_filename = self.remote_conan_home + f"/profiles/{name}"
-                sftp.putfo(BytesIO(contents.encode()), dest_filename)
-        except:
-            raise ConanException("Unable to copy profiles to remote")
-        finally:
-            sftp.close()
+    def _sync_conan_config(self):
+        # Transfer conan config to remote
+        self.remote_conn.put_dir(
+            self.conan_api.config.home(),
```

Comment:
What folder is this sending, exactly? I would probably have something more explicit rather than anything by exclusion -
We could stand simple and just sync the profiles folder (maybe the remotes and maybe global.conf) 

we should probably think ahead - there's a chance we'll have to be more selective (any configs that have paths that are relevant to the "local" machine will not work on the remote runner!)